pnpm-lock.yaml

```diff
@@ -1,18 +1,22 @@
 lockfileVersion: '6.0'
 
+settings:
+  autoInstallPeers: true
```

Comment:
This is what is causing the errors on CI. Pnpm will set `--frozen-lockfile` when running in CI and with `autoInstallPeers: true`, it will likely install things the lock files doesn't have.

Did you add this on purpose? How did it get added?

Reply:
I didn't add this on purpose, hmm. And blowing away the lockfile and re-running keeps this setting.

Reply:
Did you intentionally update the `package.json` pnpm version? I wonder if that is slipping it in. 

https://github.com/vercel/commerce/pull/1209/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R3

Reply:
I tried to intentionally update it to fix it, with no effect.